sql/sql_explain.cc

```diff
@@ -351,9 +351,9 @@ int print_explain_row(select_result_sink *result,
                       mem_root);
   
   /* 'possible_keys' */
+  StringBuffer<64> possible_keys_buf;
   if (possible_keys && !possible_keys->is_empty())
   {
-    StringBuffer<64> possible_keys_buf;
```

Comment:
Is that related to this PR? I doubt UBSAN will catch use-after-scope bugs at all.